os/sys/cc.h

```diff
@@ -49,6 +49,16 @@
 
 #ifdef __GNUC__
 
+#ifndef CC_CONF_ASSUME
+  #ifdef __clang__
+    #define ASSUME(c) do { __builtin_assume((c)); } while(0)
+  #elif __GNUC__ >= 13
+    #define ASSUME(c) __attribute__((__assume__(c)))
+  #else
+    #define ASSUME(c) do { if (!(c)) __builtin_unreachable(); } while(0)
```

Comment:
Should these not set `CC_CONF_ASSUME` in the same style as the other definitions in this file?

Reply:
Fixed.